win-odbc64/odbcclient/Drvr35Res/Drvr35Res.rc

```diff
@@ -93,7 +93,7 @@ BEGIN
             VALUE "FileDescription", "TRAF ODBC Driver Resource DLL"
             VALUE "FileVersion", "1.3.0.0"
             VALUE "InternalName", "TRAF ODBC Driver Resource DLL"
-            VALUE "LegalCopyright", "?Copyright 2015 Apache Trafodion"

```

Comment:
Why does the copyright say, "Apache Trafodion" instead of, say, "Apache Software Foundation"?